src/join_check/captcha/math_question.rs

```diff
@@ -38,15 +36,15 @@ impl Display for Operators {
     }
 }
 
-#[derive(Clone)]
-pub struct Question {
+#[derive(Clone, Copy)]
+pub struct MathQuestion {
     num1: u8,
     operator: Operators,
     num2: u8,
 }
 
-impl Question {
-    pub fn generate_question() -> (Self, [Answer; 4]) {
+impl MathQuestion {
+    pub fn generate_question() -> (Self, [u8; 4]) {
```

Comment:
Not a fan of Type Aliasing? 😄 
Let me say you that by using `Answer` everyone can at one glance see that you are producing one question and 4 answers. but without it, it will be a little harder to understand at call site.